src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -110,41 +110,57 @@
     margin-top: 30px;
 }
 .modal-content .settings-group {
+    border: 1px solid var(--settings-option-border-color);
+    border-radius: 4px;
     display: flex;
     margin-bottom: 15px;
 }
-.modal-content label {
+.modal-content .settings-group label {
     cursor: pointer;
     display: flex;
     flex: 1;
     font-size: 16px;
     margin: 0;
 }
-.modal-content label input {
-    display: none;
+.modal-content .settings-group label input {
+    position: absolute;
+    clip: rect(0, 0, 0, 0);
+    pointer-events: none;
+    opacity: 0;
+}
+.modal-content .settings-group:has(input:focus-visible) {
```

Comment:
be aware that `:has` is not yet available in Firefox (they have an experimental implementation behind a flag but it is still incomplete for now).

We might decide that this is OK for now, or we might want to provide a fallback for browsers that don't support `:has`

Reply:
You are right. But I expect that Firefox adds support for it soon. Also, the Profiler is a tool for developers, so they always update their browsers quickly.